### ISSUE TITLE
Make sure focusable elements have :focus outlines

### ISSUE THREAD

**Issue by tw2113** [WebDevStudios/custom-post-type-ui#400]

**Comment by tw2113** [comment]
Help page should be good now, but want to review the other pages and make sure they're getting focused.

**Comment by tw2113** [comment]
I think we're good for the moment. Fixed up some textarea auto-selecting onfocus in the import/export area.
